### PR TITLE
Add node-labeler

### PR DESCRIPTION
**Pull request by naved001**
Thanks to @larsks for https://github.com/larsks/opf-apps/tree/feature/add-nfs-vlan

We will configure the interfaces once we have nmstate installed.

### CODE REVIEW

File: node-labeler/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
```
Comment (naved001) [inline node-labeler/namespace.yaml:2]: should the namespace go into `cluster-scope/base/core/namespaces/` ?

Comment (larsks) [reply node-labeler/namespace.yaml:2]: Yes it should. If you fix that here, make sure to update the `ocp-prod` overlay as well to include the namespace.

**Review by larsks — APPROVED** [review]

**Review by larsks — CHANGES_REQUESTED** [review]
Oh wait, we were going to move the `Namespace` resource in `cluster-resources`.

File: cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: node-labeler
+resources:
+- namespace.yaml
```
Comment (naved001) [inline cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml:5]: the namespace is in base now.

Comment (larsks) [reply cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml:5]: ...so it should probably be removed from here.

Comment (larsks) [reply cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml:5]: Duh, never mind. Sorry!

**Review by larsks — APPROVED** [review]

File: cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml (hunk 2 above)
Comment (naved001) [reply cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml:5]: I did delete `node-labeler/namespace.yaml`, this is `cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml`

**Pull request merged**